### ISSUE TITLE
Allow $unsafe(null)

### ISSUE THREAD

**Issue by hurelhuyag** [casid/jte#176]
Today I found out if I pass a null value to `$unsafe()`. It is throwing `NullPointerException`. I think it should simply print an empty string when the value is null. 

Actual exception message:
```
nested exception is gg.jte.TemplateException: Failed to render agency/index.jte, error at template/index.jte:355] with root cause
Sep 27 06:16:16 staging Proj[9429]: java.lang.NullPointerException: Cannot invoke "String.length()" because "s" is null
```

**Comment by casid** [comment]
Yeah, that should not throw but work as you describe it. I‘m currently on vacation, but will look into it next week when I‘m back at my pc.

Thank you for reporting!

**Comment by casid** [comment]
Fixed, will be part of the next jte release :-)

**Comment by casid** [comment]
@hurelhuyag I just published `2.2.1` to maven central, including this fix.
